guides/get-started-microservices-on-kubernetes/simple-microservice-part2.adoc

```diff
@@ -0,0 +1,134 @@
+= {simple-microservice-part2}
+:summary: ONELINE EXPLANATION OF THE GUIDE
+:includedir: ../_includes
+include::{includedir}/_attributes.adoc[]
+include::./_includes/_titles.adoc[]
+// you can override any attributes eg to lengthen the
+// time to complete the guide
+:prerequisites-time: 10
+
+In this guide, you will learn how-to run the Docker Image we built in link:simple-microservice-part1[{simple-microservice-part1}] on Kubernetes.
+
+include::{includedir}/_prerequisites.adoc[]
+* You have completed link:simple-microservice-part1.adoc[Create a JAX-RS application using Bootable Jar - part one: bare metal]
```

Comment:
It would deserve to add step with link to guide how to install Kubernetes intance locally which would be used in next steps. I think it's quite a big thing so it might be another step in the guide.

Reply:
@mnovak1 thank you: I added a specific item to the "Prerequisites"